deepinv/physics/noise.py

```diff
@@ -37,6 +37,48 @@ def forward(self, x):
         return x + torch.randn_like(x) * self.sigma
 
 
+class UniformGaussianNoise(torch.nn.Module):
```

Comment:
Atm it is only done on the noise because I've not yet tried it with other inverse problems. I do this by calling the ``__init__()`` method in [here](https://github.com/deepinv/deepinv/blob/physics_train/deepinv/physics/forward.py#L113).

Reply:
ok